articles/azure-functions/durable/durable-functions-error-handling.md

```diff
@@ -103,7 +141,23 @@ If the call to the **CreditAccount** function fails for the destination account,
 
 When you call activity functions or sub-orchestration functions, you can specify an automatic retry policy. The following example attempts to call a function up to three times and waits 5 seconds between each retry:
 
-### C#
+### Precompiled C#
+
+```csharp
+[FunctionName("TimerOrchestratorWithRetry"]
```

Comment:
Typo: this function and subsequent functions are missing the trailing `)` in the attribute constructor. This is also causing the C# syntax highlighting to get messed up.